compositions/upbound-aws-provider/sns/sns.yaml

```diff
@@ -31,7 +31,7 @@ spec:
           toFieldPath: spec.forProvider.tags
           policy:
             mergeOptions:
-              appendSlice: true
+              keepMapValues: true
```

Comment:
Thanks for making these changes. Definitely needed. In the future though, let's do it in a separate PR

Reply:
will do!